api/src/main/resources/workbench.yaml

```diff
@@ -3441,3 +3466,11 @@ definitions:
       notebookName:
         type: string
 
+  NotebookLockingMetadataResponse:
+    type: object
+    properties:
+      lastLockedBy:
+        type: string
+      lockExpirationTime:
+        type: integer
```

Comment:
It's probably worth adding a description here mentioning this is ms from the unix epoch (I assume it's ms).